dace/transformation/dataflow/reduce_expansion.py

```diff
@@ -183,7 +183,7 @@ def expand(self, sdfg: SDFG, graph: SDFGState, reduce_node):
                 LocalStorage.node_a: nsdfg.sdfg.nodes()[0].nodes().index(inner_exit),
                 LocalStorage.node_b: nsdfg.sdfg.nodes()[0].nodes().index(outer_exit)
             }
-            nsdfg_id = nsdfg.sdfg.sdfg_list.index(nsdfg.sdfg)
```

Comment:
I think this behavior was done on purpose (since the cfg list wasn’t updated yet or something), but it passes the tests so far..

Reply:
Reverted for safety